eng/tools/azure-sdk-tools/azpysdk/apistub.py

```diff
@@ -0,0 +1,138 @@
+import argparse
+import os
+import sys
+
+from typing import Optional, List
+from subprocess import CalledProcessError
+
+from .Check import Check
+from ci_tools.functions import install_into_venv, find_whl
+from ci_tools.scenario.generation import create_package_and_install
+from ci_tools.variables import discover_repo_root, set_envvar_defaults
+from ci_tools.logging import logger
+from ci_tools.parsing import ParsedSetup
+
+REPO_ROOT = discover_repo_root()
+MAX_PYTHON_VERSION = (3, 11)
+
+
+def get_package_wheel_path(pkg_root: str, out_path: Optional[str]) -> tuple[str, Optional[str]]:
```

Comment:
`create_package_and_install` _does this_ for you rn. I think the missing piece is that you don't want to install it right? You just want to get the reference to the wheel to process it.  This might be the shortest path let me dig a little further in existing code before approving.